openstack/nova/templates/bigvm-deployment.yaml

```diff
@@ -36,38 +36,30 @@ spec:
         {{- end }}
     spec:
 {{ tuple . "nova" "bigvm" | include "kubernetes_pod_anti_affinity" | indent 6 }}
-{{ include "utils.proxysql.pod_settings" . | indent 6 }}
+      {{- include "utils.proxysql.pod_settings" . | indent 6 }}
       terminationGracePeriodSeconds: {{ .Values.defaults.default.graceful_shutdown_timeout | add 5 }}
       hostname: nova-bigvm
+      initContainers:
+      {{- tuple . (dict "service" (print (include "nova.helpers.cell01_services" .) (include "nova.helpers.cell2_services" .))) | include "utils.snippets.kubernetes_entrypoint_init_container" | indent 6 }}
```

Comment:
This renders as `value: nova-api-mariadb,nova-mariadb,nova-rabbitmqnova-cell-vc-b-1-mariadb,nova-cell2-rabbitmq` i.e. it's missing a `,` between the different cell services.

Reply:
Fixed by creating a snippet for it.